packages/components/src/components/contract-card/contract-card-items/__tests__/contract-update-form.spec.tsx

```diff
@@ -50,7 +50,7 @@ describe('ContractUpdateForm', () => {
         setCurrentFocus: jest.fn(),
         status: 'profit',
         toggleDialog: jest.fn(),
-    };
+    } as unknown as React.ComponentProps<typeof ContractUpdateForm>;
```

Comment:
Why are we making it as unknown?

Reply:
Because if I left previous version, then there will be a type error as contract has not expected type.
<img width="1036" alt="Screenshot 2023-11-14 at 08 46 16" src="https://github.com/binary-com/deriv-app/assets/121025168/66dbbbb9-a0e3-4516-9e88-a2a679dbac96">

Reply:
If i remove unknown, I'll still have TS error and TS suggest itself to convert via unknown, like in this case:https://github.com/binary-com/deriv-app/pull/11365/files#r1392109598

Reply:
@balakrishna-deriv i believe we should consider as unknown as acceptable for test files because we don't need to use real objects, especially complicated ones with all their multiple properties that don't make any difference for the functionality of our component. Do you agree?

Reply:
I think it's fine for now. We can think about refactoring it later.